src/slurm_plugin/cluster_event_publisher.py

```diff
@@ -775,3 +829,32 @@ def _describe_instance(instance):
             if instance
             else None
         )
+
+    @staticmethod
+    def _job_info_supplier(job: SlurmJobInfo) -> Iterator:
+        yield {
```

Comment:
As commented above, I think you should iterate over the job list in here, yielding the detail object on every iteration.